docs/text/feature_extraction_settings.rst

```diff
@@ -8,8 +8,9 @@ function and select only the relevant features later. Then you can call the func
 `kind_to_calculation_settings_mapping` object, which means you are using the default options (which will use all
 feature calculators in this package).
 
-But if you want to dig deeper into your data and use custom settings for the feature extractors, you
-have to use a custom settings object, like shown here:
+After digging deeper into your data, you maybe want to calculate more of a certain type of feature and less of another
```

Comment:
I do not like the "more" and "less". Maybe better "with other/more parameter combinations".